src/circuit/gadget/utilities.rs

```diff
@@ -84,3 +84,13 @@ where
 
     Ok(CellValue::new(cell, copy.value))
 }
+
+/// Returns an expression checking that `value` is in the range `low..=high`.
```

Comment:
I'm not sure we actually need this any more, given the technique that multiplies by 2<sup>10-s</sup>.

Reply:
The helper function `utilities::range_check()` has been removed.